x/nodes/keeper/valStateChanges.go

```diff
@@ -200,6 +201,18 @@ func (k Keeper) ValidateEditStake(ctx sdk.Ctx, currentValidator, newValidtor typ
 	if diff.IsNegative() {
 		return types.ErrMinimumEditStake(k.codespace)
 	}
+
+	if k.Cdc.IsAfterNamedFeatureActivationHeight(ctx.BlockHeight(), codec.RSCALKey) && k.Cdc.IsAfterNamedFeatureActivationHeight(ctx.BlockHeight(), codec.VEDITKey) {
+		//check that we are bumping up into a new bin or are above the cieling, if not return an error
+		if amount.LT(k.ServicerStakeWeightCeiling(ctx)) {
+			//grab the bin and check if we are in a new bin
+			flooredNewStake := amount.Sub(amount.Mod(k.ServicerStakeFloorMultiplier(ctx)))
+			if flooredNewStake.LTE(currentValidator.StakedTokens) {
```

Comment:
Bumping stake in the same bin is a desired behaviour, because a node might be getting slashed and they might be wanting to top up that stake to avoid falling to the lower bin.

Reply:
The purpose of VEDIT is to prevent continually topping up stake resulting in bloating so this would break that use case. Could add a guard to allow it if within 10pokt of dropping to the next bin. 

If it was LT only here then this would already be covered as you can't lower stake anyway.

Reply:
You are correct! I amend my assessment and consider this resolved.